src/tests/ftest/dfuse/pil4dfs_fio.yaml

```diff
@@ -0,0 +1,75 @@
+hosts:
+  test_servers: 2
+  test_clients: 1
+
+timeout: 600
+
+server_config:
+  name: daos_server
+  engines_per_host: 2
+  engines:
+    0:
+      pinned_numa_node: 0
+      fabric_iface: ib0
+      fabric_iface_port: 31317
+      log_file: daos_server0.log
+      storage:
+        0:
+          class: dcpm
+          scm_list: ["/dev/pmem0"]
+          scm_mount: /mnt/daos0
+    1:
+      pinned_numa_node: 1
+      fabric_iface: ib1
+      fabric_iface_port: 31417
+      log_file: daos_server1.log
+      storage:
+        0:
+          class: dcpm
+          scm_list: ["/dev/pmem1"]
+          scm_mount: /mnt/daos1
+
+pool:
+  size: 90%
+
+container:
+  type: POSIX
+  control_method: daos
+
+test_pil4dfs_vs_dfs:
+  bw_deltas:
+    read: 10
+    write: 10
+
+fio:
```

Comment:
i may have missed it, but what is the numjobs set to and why is not included here?

Reply:
The numjobs are automatically calculated according to the number of core available on the client node.
This number is retrieved thanks to the class CpuArchitecture defined in the file src/tests/ftest/util/cpu_utils.py.
This last class is mostly a wrapper around the lscpu command.
The list of cpus to use is defined in the following part of the code https://github.com/daos-stack/daos/blob/c22aa428ffb5576c7da2db95b7587d52184b7ad6/src/tests/ftest/dfuse/pil4dfs_fio.py#L45

At the end, there is one job per real core.